packages/framework/src/Framework/Features/Blogging/Models/PostAuthor.php

```diff
@@ -87,26 +86,38 @@ public function __construct(string $username, ?string $name = null, ?string $web
         $this->socials = $socials;
     }
 
+    /**
+     * Create a new Post Author instance from an array of data.
+     *
+     * @param  array{username?: string, name?: string, website?: string, bio?: string, avatar?: string, socials?: array<string, string>}  $data
+     */
+    public static function create(array $data): PostAuthor
```

Comment:
Check if we should use `create` or `make` to match our conventions. Using `make` may be beneficial to differentiate from the `Author` facade method with very different semantics.